### ISSUE TITLE
misleading error message in creating source

### ISSUE THREAD

**Issue by jakejh** [DevDataPlatform/DDP_backend#291]
When editing a surveycto source, if I specify a form id that does not exist, I get the error message "Failed to connect - please check your credentials." To me this implies an incorrect username or password.

**Comment by fatchat** [comment]
yes we don't know why the connection failed its test, not in general because it depends on the connector's connection-checking logic

@nairabhishek73 can we change the message to "failed to connect, it might be your credentials but we don't know"

**Comment by nairabhishek73** [comment]
For now we can do- "Failed to connect- please check details entered"

Is there no way for us to identify where the problem is (FormID vs creds) @fatchat ?

**Comment by fatchat** [comment]
not in general no... but we could choose a set of connectors e.g. the common ones our clients use and write log-parsing logic to pull the reasons out of airbyte's logs
